pysrc/bytewax/inputs.py

```diff
@@ -30,6 +30,7 @@ def tumbling_epoch(
     epoch_length: Any,
     time_getter: Callable[[Any], Any] = lambda _: datetime.datetime.now(),
     epoch_0_start_time: Any = None,
```

Comment:
Also not sure if it's helpful here, but another trick you could use is to make the start time constant between runs of the dataflow. You could pick some fixed value that you know is earlier than any data timestamp. Then the epoch will be re-calculated to be the same each time.